tests/eval-compare/builtins.appendContext.nix

```diff
@@ -0,0 +1,28 @@
+let
```

Comment:
Why do we need this test when we have the upstream one?

Reply:
Oh, good point.  @jwiegley requested it and I guess I  momentarily forgot that we have NixLanguageTests.